src/components/step-sequence/step-sequence.test.js

```diff
@@ -211,4 +146,87 @@ context("Testing StepSequence component", () => {
       }
     );
   });
+
+  describe("Accessibility tests for StepSequence component", () => {
+    it.each(["horizontal", "vertical"])(
+      "should check %s orientation for accessibility tests",
+      (orientation) => {
+        CypressMountWithProviders(
+          <StepSequenceComponent orientation={orientation} />
+        );
+        cy.checkAccessibility();
+      }
+    );
+
+    describe("check StepSequenceItem component for accessibility tests", () => {
+      it("should check StepSequenceItem with children for accessibility tests", () => {
+        CypressMountWithProviders(<StepSequenceItemCustom />);
+        cy.checkAccessibility();
+      });
+
+      it.each([["-100"], ["0"], ["999"], testData[0], testData[1]])(
+        "should check StepSequenceItem with indicator set to %s for accessibility tests",
+        (indicator) => {
+          CypressMountWithProviders(
+            <StepSequenceItemCustom status="incomplete" indicator={indicator} />
+          );
+          cy.checkAccessibility();
+        }
+      );
+
+      it.each(testData)(
+        "should check StepSequenceItem with ariaLabel set to %s for accessibility tests",
+        (ariaLabel) => {
+          CypressMountWithProviders(
+            <StepSequenceItemCustom aria-label={ariaLabel} />
+          );
+          cy.checkAccessibility();
+        }
+      );
+
+      it.each(["complete", "current", "incomplete"])(
+        "should check StepSequenceItem with status set to %s for accessibility tests",
+        (status) => {
+          CypressMountWithProviders(<StepSequenceItemCustom status={status} />);
+          cy.checkAccessibility();
+        }
+      );
+
+      it.each(testData)(
+        "should check StepSequenceItem with hiddenCompleteLabel set to %s for accessibility tests",
+        (hiddenCompleteLabel) => {
+          CypressMountWithProviders(
+            <StepSequenceItemCustom
+              status="complete"
+              hiddenCompleteLabel={hiddenCompleteLabel}
+            />
+          );
+          cy.checkAccessibility();
+        }
+      );
+
+      it.each(testData)(
+        "should check StepSequenceItem with hiddenCurrentLabel set to %s for accessibility tests",
+        (hiddenCurrentLabel) => {
+          CypressMountWithProviders(
+            <StepSequenceItemCustom
+              status="current"
+              hiddenCurrentLabel={hiddenCurrentLabel}
+            />
+          );
+          cy.checkAccessibility();
+        }
+      );
+
+      it.each(["complete", "current", "incomplete"])(
+        "should check StepSequenceItem with hideIndicator prop and status set to %s for accessibility tests",
+        (status) => {
+          CypressMountWithProviders(
+            <StepSequenceItemCustom status={status} hideIndicator />
+          );
+          cy.checkAccessibility();
+        }
+      );
+    });
+  });
```

Comment:
You could add test cases for the stories in stepsequence.stories.tsx:
Vertical
WithHiddenIndicators
ResponsiveExample

Reply:
Hi @stephenogorman - Looks like the stories you have mentioned is already there in step-sequence.stories.tsx file. If still I need to made some changes please let me know.